pages/guide.py

```diff
@@ -190,7 +190,7 @@ def process_post_data(self, **kwargs):
 
     # TODO(jrobbins): Make this be /feature/ID after ability to edit
     # from the feature detail page is complete.
-    redirect_url = '/guide/edit/' + str(key.integer_id())
+    redirect_url = '/guide/editall/' + str(key.integer_id()) + '#rollout1'
```

Comment:
Does this mean that the user will be redirected to the editall page when creating a stage on the feature detail page?

Reply:
No adding a new stage keeps you on the same page, it is only when you create a new enterprise feature that it brings you from the new feature page to the edit all page.